### PR TITLE
Drrr.com | Durarara!! Chat Presence

### PR DESCRIPTION
**Pull request by DawkinNguyen**
This is my work for drrr.com

### CODE REVIEW

File: websites/D/Durarara/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "author": {
+    "name": "Dawkin Nguyen",
+    "id": "414274321779916800"
+  },
+  "service": "Durarara!!",
+  "description": {
+    "en": "Durarara!! fan community, DOLLARS Chatroom."
+  },
+  "url": "drrr.com",
+  "version": "1.0.1",
```
Comment (Bas950) [inline websites/D/Durarara/dist/metadata.json:11]: version 1.0.0 on release

File: websites/D/Durarara/presence.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+var presence = new Presence({
+  clientId: "712269360206708766"
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/D/Durarara/presence.ts:5]: dont use var, use const instead

File: websites/D/Durarara/dist/metadata.json (hunk 1 above)
Comment (DawkinNguyen) [reply websites/D/Durarara/dist/metadata.json]: Ok

File: websites/D/Durarara/presence.ts (hunk 2 above)
Comment (DawkinNguyen) [reply websites/D/Durarara/presence.ts]: Ok

File: websites/D/Durarara/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "Dawkin Nguyen",
+    "id": "414274321779916800"
+  },
+  "service": "Durarara!!",
```
Comment (Bas950) [inline websites/D/Durarara/dist/metadata.json:6]: service name must be the same as folder name

Comment (DawkinNguyen) [reply websites/D/Durarara/dist/metadata.json]: ok

**Pull request merged**